test/functional/chrome-e2e-specs.js

```diff
@@ -0,0 +1,32 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import AndroidDriver from '../..';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+const capabilities = {
+  "browserName": "chrome",
+  "avd": "Nexus_5_API_25",
```

Comment:
should we hardcode the avd name here or take it from process.env? What do we do in other tests?